src/statscollector/DefaultStatsCollector.ts

```diff
@@ -440,43 +444,18 @@ export default class DefaultStatsCollector implements StatsCollector {
       return;
     }
     const rawMetricReports: RawMetricReport[] = [];
-    if (!this.browserBehavior.requiresPromiseBasedWebRTCGetStats()) {
-      // @ts-ignore
-      this.audioVideoController.rtcPeerConnection.getStats(
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        (res: any) => {
-          // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          res.result().forEach((report: any) => {
-            const item: { [name: string]: StatsReportItem } = {};
-            report.names().forEach((name: string) => {
-              item[name] = report.stat(name);
-            });
-            item.id = report.id;
-            item.type = report.type;
-            item.timestamp = report.timestamp;
-            rawMetricReports.push(item);
-          });
-          this.handleRawMetricReports(rawMetricReports);
-        },
-        // @ts-ignore
-        (error: Error) => {
-          this.logger.error(error.message);
-        }
-      );
-    } else {
-      // @ts-ignore
-      this.audioVideoController.rtcPeerConnection
-        .getStats()
-        .then((report: RTCStatsReport) => {
-          report.forEach((item: StatsReportItem) => {
-            rawMetricReports.push(item);
-          });
-          this.handleRawMetricReports(rawMetricReports);
-        })
-        .catch((error: Error) => {
-          this.logger.error(error.message);
+    // @ts-ignore
+    this.audioVideoController.rtcPeerConnection
+      .getStats()
```

Comment:
Are you sure this is safe to do? Are we going through all existing legacy googStats and verify they have a mapping to new one?

Reply:
Yes, I will make sure to have all the mappings for the lagacy googStats. Thanks for helping me with the checking.